src/scala/com/github/johnynek/bazel_deps/MakeDeps.scala

```diff
@@ -193,19 +193,19 @@ object MakeDeps {
             model.getOptions.getResolvers.collect { case e: MavenServer => e },
             resolverCachePath)
         )
-      case ResolverType.Coursier =>
+      case c: ResolverType.Coursier =>
         val ec = scala.concurrent.ExecutionContext.Implicits.global
         import scala.concurrent.duration._
         resolve(
           model,
-          new CoursierResolver(model.getOptions.getResolvers, ec, 3600.seconds, resolverCachePath)
+          new CoursierResolver(model.getOptions.getResolvers, c.getHashInHttpHeaders, ec, 3600.seconds, resolverCachePath)
         )
       case g: ResolverType.Gradle =>
         val ec = scala.concurrent.ExecutionContext.Implicits.global
         import scala.concurrent.duration._
 
         lazy val coursierResolver =
-          new CoursierResolver(model.getOptions.getResolvers, ec, 3600.seconds, resolverCachePath)
+          new CoursierResolver(model.getOptions.getResolvers, false, ec, 3600.seconds, resolverCachePath)
```

Comment:
same comment as above: if this works, shouldn't true be the default? Why would you set false? Maybe if you know with certainty your http service doesn't give out the headers?